tests/testing_utils.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, scene_object, module_tested, caching_needed=False):
         if caching_needed:
             config['text_dir'] = os.path.join(
```

Comment:
this one too, no?

Reply:
That's the issue. One uses `config `and the other `file_writer_config`.